rustls/src/conn.rs

```diff
@@ -1049,15 +1047,11 @@ impl CommonState {
             Limit::No => payload.len(),
         };
 
-        let mut plain_messages = VecDeque::new();
-        self.message_fragmenter.fragment_borrow(
+        for m in self.message_fragmenter.fragment_slice(
```

Comment:
Style nit: I dislike having multi-line expressions in block headers like `for` or `if`. How about naming the return value of `fragment_slice()` (I'd suggest just `iter`)?